docs/components/modeler/desktop-modeler/connect-to-camunda-8.md

```diff
@@ -0,0 +1,31 @@
+---
+id: connect-to-camunda-8
+title: Deploy your first diagram
+description: "Follow these steps to directly deploy diagrams and start process instances in Camunda 8."
+---
+
+Desktop Modeler can directly deploy diagrams and start process instances in Camunda 8. Follow the steps below to deploy a diagram to **Camunda 8 SaaS**. To deploy to a local installation, visit the [Camunda 8 Self-Managed guide](../../../self-managed/modeler/desktop-modeler/deploy-to-self-managed.md).
+
+1. Click the deployment icon:
+
+![deployment icon](./img/deploy-icon.png)
+
+2. Click **Camunda Platform 8 SaaS**:
+
+[//]: <> (I am assuming this will be adjusted on the UI end?)
```

Comment:
Good question. Can you sync with the Desktop Modeler team and point them to the Slack channel for renaming so they have the complete context? I don't see them represented in the spreadsheet.

For our renaming exercise, I don't see this as blocking. We should have this reflect what is in the UI, and for now, that includes "platform."

Reply:
FYI - https://github.com/camunda/camunda-platform-docs/pull/2571#issuecomment-1714032909